Source/Atj/data/mvp_scenario.json

```diff
@@ -1,5 +1,6 @@
 {
     "npcs": [
+        "player",
```

Comment:
I asked you to put this here, but on second thought, lets remove it. The "player" implicitly exists, there's no reason to require you to specify them in the scene.